app/(main)/league/[leagueId]/entry/all/page.tsx

```diff
@@ -37,6 +37,10 @@ const Entry = ({
    * @returns {Promise<void>}
    */
   const getAllEntries = async (): Promise<void> => {
+    if (!user.id || user.id === '') {
```

Comment:
I'm not sure I understand the meaning behind adding this conditional when you're supposed to be checking if a team is selected, or not selected, it feels a little out of scope. Could you please explain?

Reply:
Oh, you're right. It's redundant work and I will remove this.